packages/core/src/schema/models/documentVersions.ts

```diff
@@ -23,6 +24,7 @@ export const documentVersions = latitudeSchema.table(
     content: text('content').notNull().default(''),
     resolvedContent: text('resolved_content'),
     contentHash: text('content_hash'),
+    promptlVersion: integer('promptl_version').notNull().default(0),
```

Comment:
move drizzle changes to another PR and deploy that before merging this

Reply:
https://github.com/latitude-dev/latitude-llm/pull/629